src/main/java/de/dhbw/mh/slang/codegen/JavaBytecodeGenerator.java

```diff
@@ -1,109 +1,128 @@
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case MULTIPLY:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case DIVIDE:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case MODULO:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case LOGICAL_AND:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case LOGICAL_OR:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case COMPARE_EQUAL:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case COMPARE_UNEQUAL:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case LESS_THAN:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case GREATER_OR_EQUAL:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case GREATER_THAN:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case LESS_OR_EQUAL:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-			case POWER:{
-				throw new RuntimeException( "not yet implemented" );
-			}
-		}
-		throw new RuntimeException( "unhandled branch" );
-	}
+    @Override
+    public String visitPost(AstUnaryOperation node, String base) {
+        switch (node.OPERATOR) {
+            case NEGATIVE_SIGN: {
+                throw new RuntimeException("not yet implemented");
+            }
+            case POSITIVE_SIGN: {
+                throw new RuntimeException("not yet implemented");
+            }
+        }
+        throw new RuntimeException("unhandled branch");
+    }
 
+    @Override
+    public String visitPost(AstBinaryOperation node, String lhs, String rhs) {
+        switch (node.OPERATOR) {
+            case ADD: {
+                throw new RuntimeException("not yet implemented");
+            }
+            case SUBTRACT: {
+                throw new RuntimeException("not yet implemented");
+            }
+            case MULTIPLY: {
+                char resultChar = getBytecodeDatatypeChar(node.getDatatype());
+
+                return String.format("%s%s%cmul%n", lhs, rhs, resultChar);
+            }
+            case DIVIDE: {
+                throw new RuntimeException("not yet implemented");
+            }
+            case MODULO: {
+                throw new RuntimeException("not yet implemented");
+            }
+            case LOGICAL_AND: {
+                throw new RuntimeException("not yet implemented");
+            }
+            case LOGICAL_OR: {
+                throw new RuntimeException("not yet implemented");
+            }
+            case COMPARE_EQUAL: {
+                throw new RuntimeException("not yet implemented");
+            }
+            case COMPARE_UNEQUAL: {
+                throw new RuntimeException("not yet implemented");
+            }
+            case LESS_THAN: {
+                return String.format("%s%sif_icmpge #0_ge%niconst_1%ngoto #0_end%n#0_ge:%niconst_0%n#0_end:%n", lhs, rhs);
```

Comment:
It would be nice to have a counter for the label ID's. I will approve your PR anyway to avoid a ton of merge conflicts 😆